soroban-sdk/src/env.rs

```diff
@@ -244,6 +244,28 @@ impl Env {
     pub fn log_value<V: IntoVal<Env, RawVal>>(&self, v: V) {
         internal::Env::log_value(self, v.into_val(self));
     }
+
+    #[doc(hidden)]
+    pub fn get_ledger_version(&self) -> u32 {
+        internal::Env::get_ledger_version(self).try_into().unwrap()
+    }
+
+    #[doc(hidden)]
+    pub fn get_ledger_sequence(&self) -> u32 {
+        internal::Env::get_ledger_sequence(self).try_into().unwrap()
+    }
+
+    #[doc(hidden)]
+    pub fn get_ledger_timestamp(&self) -> u64 {
```

Comment:
Since we're just returning an int (and not some time type), can we add the unit to the name? (e.g. `get_ledger_timestamp_micros`  (I've no idea what the actual unit is).

Reply:
It's in seconds:
https://github.com/stellar/rs-soroban-sdk/blob/aa61c62a01cd7a8ba4297889945a7ddd68b46aaf/soroban-sdk/src/ledger.rs#L70-L75

Reply:
I don't feel strongly about naming it `timestamp` vs `timestamp_secs`. As a comparison in Ethereum it is named `timestamp` ([ref](https://docs.soliditylang.org/en/v0.8.15/units-and-global-variables.html#block-and-transaction-properties)).

`_secs` would answer the question though that is most often posed.

There could also be an argument for us making it a struct that exposes a `seconds` function I think.